tools/src/tensil/tools/compiler/OnnxFrontend.scala

```diff
@@ -2042,6 +2053,55 @@ class OnnxFrontend(
     outputTemp
   }
 
+  private def emitLayerMatMul(
+      context: EmitContext,
+      scheduler: Scheduler,
+      matMulProto: NodeProto
+  ): MemoryObject = {
+    context.mm.addPendingConst(
```

Comment:
Can you remind me what is the difference between addPendingConst and getOrEmitWeights?